docker/Dockerfile-prod

```diff
@@ -4,22 +4,59 @@
 
 FROM python:3.5-alpine
 
-RUN addgroup -g 1001 app && \
-    adduser -D -u 1001 -G app -s /usr/sbin/nologin app
+MAINTAINER mars@mozilla.com
+# These are unlikely to change from version to version of the container
+EXPOSE 9000
+CMD ["/usr/local/bin/uwsgi"]
+
+RUN addgroup -g 10001 app && adduser -D -u 10001 -G app -h /app app
+
+# uWSGI configuration
+ENV UWSGI_MODULE=landoapi.wsgi:app \
+	UWSGI_SOCKET=:9000 \
+	UWSGI_MASTER=1 \
+	UWSGI_WORKERS=2 \
+	UWSGI_THREADS=8 \
+    # Disable worker memory sharing optimizations.  They can cause memory leaks
+    # and issues with packages like Sentry.
+    # See https://discuss.newrelic.com/t/newrelic-agent-produces-system-error/43446
+	UWSGI_LAZY_APPS=1 \
+	UWSGI_WSGI_ENV_BEHAVIOR=holy \
+    # Make uWSGI die instead of reload when it gets SIGTERM (fixed in uWSGI 2.1)
+    UWSGI_DIE_ON_TERM=1 \
+    # Check that the options we gave uWSGI are sane
+    UWSGI_STRICT=1 \
+    # Die if the application threw an exception on startup
+    UWSGI_NEED_APP=1
 
 RUN apk --update --no-cache add \
 	sqlite
 RUN mkdir /db
 RUN chown app:app /db
 
 COPY requirements.txt /requirements.txt
-RUN pip install --no-cache -r /requirements.txt
+
+
+RUN set -ex \
```

Comment:
This stanza is standard in the official Docker image's Dockerfiles.  I've added a comment to the code explaining what it does.

The big thing it does is remove the need to explicitly state a dependency on both package XYZ and XYZ-dev in the Dockerfile, since the 'scanelf' stanza automagically keeps XYZ and removes XYZ-dev.  Otherwise I would have to keep 2 separate lists of packages in the Dockerfile, and keep them in sync, too.  Keeping one list should make the housekeeping easier.  It's fewer lines to read, understand, and maintain.